web/src/pages/app/index.tsx

```diff
@@ -31,9 +30,11 @@ const query = gql`
   }
 `;
 
-export const getServerSideProps: GetServerSideProps = requireAuthentication(
-  async (_context, user_id) => {
+export const getServerSideProps: GetServerSideProps = withPageAuthRequired({
+  getServerSideProps: async ({ req, res }) => {
+    const session = await getSession(req, res);
     const client = await getAPIServiceClient();
+    const user_id = session?.user.hasura.id;
```

Comment:
where is auth enforced for these routes? i.e. what prevents `user_id` from being null?